example/main.tf

```diff
@@ -0,0 +1,23 @@
+
+locals {
+  full_domain_name = "${var.certificate_subdomain}.${var.cloudflare_zone_name}"
+}
+
+module "certificate" {
+  source  = "silinternational/acm-certificate/aws"
+  version = "0.1.0"
+
+  certificate_domain_name = local.full_domain_name
+  cloudflare_zone_name    = var.cloudflare_zone_name
+  create_dns_validation   = var.create_dns_validation
+}
+
+module "domain" {
+  source  = "silinternational/api-gateway-custom-domain/aws"
```

Comment:
Is this a current module, or one you'll be creating?

Reply:
That's another one I plan to create (aka extract from my current local code)